mmv1/products/documentaiwarehouse/Location.yaml

```diff
@@ -0,0 +1,96 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Location'
+base_url: 'projects/{{project}}/locations/{{location}}:initialize'
```

Comment:
It looks like `{{project}}` should be changed to `{{project_number}}`, as the generated [documentation](https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-8371-old..auto-pr-8371#diff-8e71be9bedbbb37beae499fcb4a7d54bb21278c1aafe2b8096afdfe7e5a83239R87) is not accurate if `{{project}}` is used here. And`project_number` should be added as another parameter field similar to `location`

Reply:
Okay, thanks, I've changed it in Location and DocumentSchema both